index.js

```diff
@@ -54,7 +70,16 @@ function getHighestMetascore() {}
  *  getAverageIMDBRating(movies);
  *  //> 7.76
  */
-function getAverageIMDBRating() {}
+function getAverageIMDBRating(movies) {
+  let average = 0;
+  for (let i = 0; i < movies.length; i++) {
+    if (movies[i].imbdRating) {
+      average = (average + movies[i].imbdRating) / movies.length
```

Comment:
I think where this one got away from us was here. We should either be accumulating the scores by adding them to the `average` variable and then dividing by the `movies.length` during the return, or we should be adding to `average` `movies[i].imbdRating) / movies.length`, but not both which is what this line of code looks like it is doing.